hmda-quarterly-data-service/src/test/scala/hmda/quarterly/data/api/route/denials/DenialRatesByTypeLoanPurposeHomeSpec.scala

```diff
@@ -0,0 +1,20 @@
+package hmda.quarterly.data.api.route
+
+import org.scalatest.{Matchers, WordSpec}
+import hmda.quarterly.data.api.route.rates.denials.DenialRatesByTypeLoanPurposeHome
+
+
+class DenialRatesByTypeLoanPurposeHomeSpec extends WordSpec with Matchers {
+  val route = DenialRatesByTypeLoanPurposeHome.getRoute
+  val routeSummary = DenialRatesByTypeLoanPurposeHome.getSummary
+  "denial rates by type loan purpose home route" should {
+    "return the correct summary route" in {
+      assert(!routeSummary.isCompleted)
```

Comment:
Why are we asserting that it is not completed? I noticed some of the "return the correct summary route" sections in this PR are `assert(!routeSummary.isCompleted)`, but most are `assert(routeSummary.isCompleted)`.

Reply:
Agreed, resolved this by testing that the route returned the correct type of object after completion.